xrootd/client/filesystem_test.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2018 The go-hep Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build xrootd_test_with_server
```

Comment:
let's remove this build tag to ensure Travis is testing against the C++ server :)